issuance/issuance.go

```diff
@@ -684,17 +741,22 @@ func LoadChain(certFiles []string) ([]*Certificate, error) {
 	}
 
 	// Iterate over all certs except for the last, checking that their signature
-	// comes from the next cert in the list
+	// comes from the next cert in the list.
 	chain := make([]*Certificate, len(certFiles)-1)
 	for i := 0; i < len(certs)-1; i++ {
 		err = certs[i].CheckSignatureFrom(certs[i+1])
 		if err != nil {
 			return nil, fmt.Errorf("failed to verify chain: %w", err)
 		}
 		// Add each cert to the chain.
-		chain[i] = &Certificate{certs[i]}
+		ic, err := NewCertificate(certs[i])
+		if err != nil {
+			return nil, fmt.Errorf("failed to compute hashes for certificate: %w", err)
```

Comment:
```suggestion
			return nil, fmt.Errorf("failed to compute hashes for certificate: %w", err)
```

Similar to above, this code isn't responsible for knowing that the problem is with hashes; that's NewCertificate's job. Here we should wrap the error with information that includes the filename, and which entry in the chain it was.

Reply:
Simplified this to just use `issuance.LoadCertificate` above, so that there aren't two separate steps (loading from file, computing hashes) that have to be error-handled individually.